lib/src/mapbox_map.dart

```diff
@@ -98,6 +100,10 @@ class MapboxMap extends StatefulWidget {
   /// Biggest impact in android
   final bool dragEnabled;
 
+  final bool logoEnabled;
+
+  final bool attributionEnabled;
```

Comment:
please add a doc comment here to notify users that they are required to provide proper attribution to any kind of layers they show. This is true even if mapbox maps are not used - e.g. https://www.openstreetmap.org/copyright